File: src/MainService.Models/Entities/WorkSchedule.cs
namespace MainService.Models.Entities
{
    public class WorkSchedule : BaseEntity
    {
        public int DayOfWeek { get; set; }
        public TimeOnly StartTime { get; set; }
        public TimeOnly EndTime { get; set; }
        public DoctorWorkSchedule DoctorWorkSchedule { get; set; }
    }
}

File: client/src/app/_models/workSchedules/workSchedule.ts

export class WorkSchedule {
  id: number | null = null;
  startTime: string | null = null;
  endTime: string | null = null;
  dayOfWeek: number | null = null;

  constructor(init?: Partial<WorkSchedule>) {
    Object.assign(this, init);
  }
}
